src/parser/jobs/mnk/modules/Demolish.tsx

```diff
@@ -44,16 +44,18 @@ export class Demolish extends DoTs {
 		}))
 	}
 
-	protected override addClippingSuggestions(clip: DotDurations) {
+	protected override addClippingSuggestions() {
+		const demolishClipPerMinute = this.getClippingAmount(this.data.statuses.DEMOLISH.id)
+
 		this.suggestions.add(new TieredSuggestion({
 			icon: this.data.actions.DEMOLISH.icon,
 			content: <Trans id="mnk.demolish.suggestion.content">
 				Avoid refreshing <ActionLink action="DEMOLISH"/> significantly before its expiration. Unnecessary refreshes risk overwriting buff snapshots.
 			</Trans>,
 			tiers: SUGGESTION_TIERS.CLIPPING,
-			value: this.getClippingAmount(this.data.statuses.DEMOLISH.id),
+			value: demolishClipPerMinute,
 			why: <Trans id="mnk.demolish.suggestion.why">
-				You lost {this.parser.formatDuration(clip[this.data.statuses.DEMOLISH.id] ?? 0)} of Demolish to early refreshes.
+				An average of {this.parser.formatDuration(demolishClipPerMinute, 1)} seconds of <DataLink status="DEMOLISH" /> per minute lost to early refreshes.
```

Comment:
Prefer to keep the structure of the old text, something like this:
```suggestion
				You lost {this.parser.formatDuration(demolishClipPerMinute, 1)} seconds of <DataLink status="DEMOLISH" /> per minute on average to early refreshes.
```

Reply:
There's been a mix of the two formats between jobs, and I saw more jobs with the "An average of..." format so I standardized on that.  Let me know if you would prefer to revert @ayyaruq

Reply:
Applying the suggestion is fine imo. I know we like to standardise wording where possible but since this is in the job modules rather than the parent class itself I'd like to word it this way so it's clearer that the loss is the focus, especially for how important MNK's DoT is relative to other jobs.